<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="MainFrame">
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="centralwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>331</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Server Setting</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox:editable {
   
    border: 1px solid black;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PORT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 4px;
    border: 1px solid gray;
}

QLineEdit:focus {
    border: 1px solid rgb(64, 100, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openServerButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>125</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 4px;
    border: 1px solid gray;
    background-color: white;
}

QPushButton:hover {
    background-color: rgb(236, 245, 255);
    border: 1px solid rgb(64, 158, 255);
    color: rgb(64, 158, 255);
}

QPushButton:pressed {
    background-color: rgb(236, 245, 255);
    border: 1px solid rgb(64, 100, 255);
    color: rgb(64, 100, 255);
}</string>
      </property>
      <property name="text">
       <string>Open Server</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeServerButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>125</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 4px;
    border: 1px solid gray;
    background-color: white;
}

QPushButton:hover {
    background-color: rgb(236, 245, 255);
    border: 1px solid rgb(64, 158, 255);
    color: rgb(64, 158, 255);
}

QPushButton:pressed {
    background-color: rgb(236, 245, 255);
    border: 1px solid rgb(64, 100, 255);
    color: rgb(64, 100, 255);
}</string>
      </property>
      <property name="text">
       <string>Close Server</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>331</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Online Accounts</string>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>271</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
	border-radius: 8px;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>541</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Info</string>
     </property>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>461</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
